src/main/java/tetris/Controller.java

```diff
@@ -10,6 +12,41 @@
 
 public class Controller {
 
+    private class Score extends Observable {
```

Comment:
Why do you choose to put this class in the controller rather than in a separate file?